galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/internal/Zos3270ManagerImpl.java

```diff
@@ -39,18 +40,18 @@
 
 @Component(service = { IManager.class })
 public class Zos3270ManagerImpl extends AbstractManager implements IZos3270ManagerSpi {
-    protected static final String              NAMESPACE     = "zos3270";
+    protected static final String                       NAMESPACE     = "zos3270";
 
-    private static final Log                   logger        = LogFactory.getLog(Zos3270ManagerImpl.class);
+    private static final Log                            logger        = LogFactory.getLog(Zos3270ManagerImpl.class);
 
-    private IConfigurationPropertyStoreService cps;
-    private IDynamicStatusStoreService         dss;
+    private IConfigurationPropertyStoreService          cps;
+    private IDynamicStatusStoreService                  dss;
 
-    private IZosManagerSpi                     zosManager;
+    private IZosManagerSpi                              zosManager;
 
-    private ArrayList<Zos3270TerminalImpl>     terminals     = new ArrayList<>();
+    private HashMap<Zos3270TerminalImpl, Boolean>       terminals     = new HashMap<>();
```

Comment:
Really?   you are putting configuration information in a hashmap?    change the constructor of Zos3270TerminalImp to accept autoconnect and then query the impl to see if it wants autoconnect at startup.